doc/source/index.rst

```diff
@@ -19,58 +19,3 @@ The Drizzlepac package is a collection packages for aligning and combining astro
     explanation
     parameters
     reference
-
-
-
-
-.. DrizzlePac Release Notes
-.. ------------------------
-.. The code for this package gets released through a number of methods: namely,
-.. the use of the package for pipeline and archive processing of ACS and WFC3
-.. data, SSB's semi-annual public release of the stsci_python package, and a
-.. weekly beta release of the development version.  The following notes
-.. provide some details on what has been revised for each version.
-
-
-.. Image Registration Tasks
-.. ------------------------
-.. A number of tasks have been developed to support the registration
-.. of images.  This includes documentation for the replacement task
-.. for IRAF's ``tweakshifts``, currently named ``TweakReg``, along
-.. with tasks for updating the WCS in HST images and performing
-.. photometry equalization for WFPC2 data.
-
-.. In addition, several tasks have been developed to perform
-.. coordinate transformations that take into account the full
-.. distortion model specified in HST image headers.
-
-
-
-.. Reproducing Pipeline Processing
```

Comment:
Just this little section "Reproducing Pipeline Processing" should not be lost.  Drizzlepac actually has three major "modes": traditional pipeline, HAP SVM, and HAP MVM.  Of course, now all three are the new world pipeline.